platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/module/telemetry/EventMetaData.java

```diff
@@ -0,0 +1,65 @@
+package com.mapbox.mapboxsdk.module.telemetry;
+
+import android.app.ActivityManager;
+import android.content.Context;
+import android.os.Build;
+import android.os.Bundle;
+
+import android.util.DisplayMetrics;
+import android.view.Display;
+import android.view.WindowManager;
+
+import com.google.gson.JsonObject;
+import com.mapbox.mapboxsdk.Mapbox;
+import java.util.Locale;
+
+
+public class EventMetaData {
+
+  private static String METADATA_KEY = "metadata";
+  private static JsonObject eventMetaData = null;
```

Comment:
doesn't need to be static, each perf event needs to be able to provide different metadata